filestore/s3_test.go

```diff
@@ -123,7 +138,8 @@ func (t *TestSuite) TestConstructFailBadBucketName() {
 }
 
 func constructFail(t *TestSuite, client *s3.S3, min *minio.Client, bucket string, expected error) {
-	fstore, err := NewS3FileStore(client, min, bucket, false)
+	httpClient := httpClient()
```

Comment:
I probably would've just inlined the method call for these cases where it's not reused, but meh

Reply:
meh